packages/scripts/src/upgrade-tests/voting2/2_Verify.ts

```diff
@@ -0,0 +1,60 @@
+// This script verify that the upgrade was executed correctly.
```

Comment:
before running this you need to run `simulateVote` to execute the vote process, right?

Reply:
Correct, it's documented here: https://github.com/UMAprotocol/protocol/blob/e3cabaaa55e947aa42545aa92c6ce7a55edbb02d/packages/scripts/src/upgrade-tests/voting2/readme.md